### PR TITLE
bone-resume:0.2.0

### PR DESCRIPTION
**Pull request by zrr1999**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: BoneResume is a resume template for [Typst](https://github.com/typst/typst).

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/bone-resume/0.2.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  show par: set block(spacing: 0.65em)
+
+  align(center)[
+    #block(text(weight: 700, 1.75em, title))
+  ]
+  set par(justify: true)
+  body
+}
+
+#let primary_achievement(name, decs, contrib) = {
```
Comment (elegaanz) [inline packages/preview/bone-resume/0.2.0/lib.typ:28]: To be consistent with other functions, you may want to rename that one to `primary-achievement`.

**Review by elegaanz — COMMENTED** [review]
One small naming suggestion, otherwise it looks good.

File: packages/preview/bone-resume/0.2.0/lib.typ (hunk 1 above)
Comment (zrr1999) [reply packages/preview/bone-resume/0.2.0/lib.typ]: Okay, it has been modified, thank you.

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**